src/node/historical_queries_adapter.cpp

```diff
@@ -10,6 +10,58 @@
 #include "node/rpc/network_identity_subsystem.h"
 #include "node/tx_receipt_impl.h"
 
+#include <t_cose/t_cose_sign1_sign.h>
+
+namespace
+{
+  static constexpr int64_t LEAF_LABEL = 404; // TBD
```

Comment:
Adding the link to https://github.com/ietf-scitt/draft-birkholz-cose-cometre-ccf-profile in comment is probably good.